jest.config.js

```diff
@@ -1,3 +1,4 @@
 module.exports = {
-  preset: '@stoplight/scripts',
+  preset: 'ts-jest',
+  testEnvironment: 'node',
```

Comment:
Does this fix something?

Reply:
The current preset does not work anymore

![image](https://user-images.githubusercontent.com/1416224/76590679-f6c52a80-64bb-11ea-9814-e1424f3e3cb4.png)